src/iptvsimple/Epg.h

```diff
@@ -24,6 +24,7 @@
 #include "kodi/libXBMC_pvr.h"
 
 #include "Channels.h"
+#include "Settings.h"
```

Comment:
I assume you need this include for `ADDON_DATA_BASE_DIR`?

Reply:
Yes

Reply:
okay.